ember-composable-helpers/package.json

```diff
@@ -72,32 +72,43 @@
     "ember-functions-as-helper-polyfill": "^2.1.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.0",
-    "@babel/plugin-proposal-decorators": "^7.25.9",
-    "@babel/plugin-transform-class-properties": "^7.25.9",
-    "@babel/plugin-transform-typescript": "^7.26.3",
-    "@babel/runtime": "^7.26.0",
-    "@embroider/addon-dev": "^4.1.0",
-    "@glint/core": "^1.5.0",
-    "@glint/environment-ember-loose": "^1.5.0",
-    "@glint/environment-ember-template-imports": "^1.5.0",
-    "@glint/template": "^1.5.0",
-    "@rollup/plugin-babel": "^6.0.3",
-    "@tsconfig/ember": "^3.0.2",
+    "@babel/core": "^7.26.9",
+    "@babel/eslint-parser": "^7.26.8",
+    "@babel/plugin-transform-typescript": "^7.26.8",
+    "@babel/runtime": "^7.26.9",
+    "@embroider/addon-dev": "^7.1.1",
+    "@eslint/js": "^9.21.0",
+    "@glint/core": "^1.5.2",
+    "@glint/environment-ember-loose": "^1.5.2",
+    "@glint/environment-ember-template-imports": "^1.5.2",
+    "@glint/template": "^1.5.2",
+    "@rollup/plugin-babel": "^6.0.4",
+    "@tsconfig/ember": "^3.0.9",
     "@types/rsvp": "^4.0.9",
-    "@typescript-eslint/eslint-plugin": "^6.14.0",
-    "@typescript-eslint/parser": "^6.14.0",
+    "babel-plugin-ember-template-compilation": "^2.3.0",
     "concurrently": "^9.1.2",
     "ember-source": "~5.8.0",
     "ember-template-lint": "^6.0.0",
-    "eslint-plugin-node": "^10.0.0",
+    "eslint": "^9.21.0",
+    "eslint-config-prettier": "^10.0.2",
+    "eslint-plugin-ember": "^12.5.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-n": "^17.16.1",
+    "globals": "^16.0.0",
     "prettier": "^3.5.2",
     "prettier-plugin-ember-template-tag": "^2.0.4",
     "rollup": "^3.29.1",
     "rollup-plugin-copy": "^3.5.0",
     "typescript": "^5.3.3",
+    "typescript-eslint": "^8.25.0",
     "webpack": "^5.88.1"
   },
+  "peerDependencies": {
```

Comment:
can you remove this object?

we also support 3.28+ 

https://github.com/NullVoxPopuli/ember-composable-helpers/?tab=readme-ov-file#compatibility

<3